physicsnemo/models/module.py

```diff
@@ -177,10 +177,6 @@ def _override_args(
                 raise ValueError(
                     f"Argument '{key}' cannot be overridden for " f"{cls.__name__}."
                 )
-            if key not in args:
```

Comment:
Just curious, would it make sense to replace exception with logging a warning rather than removing this check altogether?